### ISSUE TITLE
[Bug]: "Move to Trash" bypasses the Trash bin and deletes immediately

### ISSUE THREAD

**Issue by lindenstruth** [mikekazakov/nimble-commander#440]
### Confirmation

- [X] I have read the contributing guidelines

### Version of Nimble Commander

v1.6.1 running on macOS 14.7.1

### Distribution Type

Downloaded from GitHub (nightly and test workflows)

### Expected Result

Files and folders are being moved to the Trash bin and can be restored, if accidentally moved.

### Actual Result

The files and folders are just gone. They're being deleted immediately by bypassing the Trash bin.

### Steps to reproduce

Move something to the Trash using the keyboard shortcut CMD+Backspace or the respective main menu entry

**Comment by lindenstruth** [comment]
Seems this issue is somehow related to an external volume's file system; doesn't happen on the system volume. So I'm closing this for now since it's most likely not NC's fault at all.

**Comment by lindenstruth** [comment]
For the record: it seems something was wrong with my ext. volume's trash can (an APFS volume as a matter of fact). Nothing I moved there did actually end up there. But once I moved an empty folder there using the macOS Finder, all those files & folders which I previously moved there using NC (or ForkLift 3 which showed the same behavior) and which were supposedly lost suddenly all reappeared out of nowhere.

Haven't had this issue ever since...
